apple/testing/default_runner/ios_xctestrun_runner.template.sh

```diff
@@ -524,6 +532,15 @@ else
     || test_exit_code=$?
 fi
 
+post_action_binary=%(post_action_binary)s
+if [[ -x "${post_action_binary:-}" ]]; then
```

Comment:
Why do we need to check this? Will it ever not be true?

Reply:
Just a way to safeguard against a bad actor. Also at one point this code accepted an empty string instead of the fixed `"/usr/bin/true"` defined in the rule. I'll remove if you prefer.

Reply:
Yeah, let’s remove it.